Enterprise-CMCS/managed-care-review
# MCR-4322 Fix mailto link to email help desk

## Summary
This tickets fixes mailto link that we use to contact help desk. To make this type of bug unlikely in the future, I tried to unify the component that links to the help desk.

Consulting design,  the error alert messaging is now  using  "email the help desk" everywhere  instead of "let us know" and a combination of other text phrases.  Also  changed ErrorAlert UI and props. We now differentiate between `message` (the core error message) and `remediation` which is an optional secondary  area of the alert where the steps the user needs to take to fix the issue is discussed.  This makes all error alerts more similar in impleentation to our generic api error, which is the most common alert used in the application. 


#### Related issues
https://jiraent.cms.gov/browse/MCR-4322

#### Screenshots
Error alerts with new "email help desk" langauge
![Screenshot 2024-08-22 at 11 41 53 PM](https://github.com/user-attachments/assets/4986636f-7610-47ba-a8d0-ce8038e121f0)


#### Test cases covered
`displays message with bold text when withEmphasis is used`
`displays email support link mailto link with default remediation`



## QA guidance
Try triggering some error states in the application, make sure error alerts look alright and that mailto link opens email.
